frontend/src/pages/releases/releases.tsx

```diff
@@ -10,6 +10,14 @@ type ReleaseNote = {
 };
 
 const releaseNotes: ReleaseNote[] = [
+  // Add more releases below
+  {
+    title: 'Optimizing Bot Traffic Handling for OG Tags: a Long Journey',
```

Comment:
How about "Optimizing Bot Traffic Handling for Link Previews: a Long Journey"

(may make it sound slightly more accessible for people who don't know what OG tags are) would change title in markdown file too obv